src/hard/moveZeros.ts

```diff
@@ -7,5 +7,24 @@
 export const moveZeros = (
   arr: (string | number | boolean | null | object | Array<[]>)[]
 ) => {
-  return false;
+  let arrCopy = [...arr];
```

Comment:
I was thinking about this today.. A much more straight forward solution would be to count up the zeroes, split the string on the zeroes, rejoin the string and add the zeroes to the end.